src/model/Document.ts

```diff
@@ -26,6 +26,7 @@ export default class Document extends Resource implements DocumentData {
 
     constructor(data: DocumentData) {
         super(data);
+        this.vocabulary = data.vocabulary;
```

Comment:
ltgm.

Just a note - to me, we use lots of questionmarked fields in our API (e.g. the field vocabulary? in this class), which make validation difficult.

Reply:
Yeah, well there can be standalone documents which do not have a vocabulary, so I don't see much choice (apart from having a subclass which would contain the vocabulary reference).